cpp/benchmarks/sort/sort.cpp

```diff
@@ -60,4 +68,5 @@ NVBENCH_BENCH_TYPES(bench_sort, NVBENCH_TYPE_AXES(Types))
   .add_int64_axis("stable", {0, 1})
```

Comment:
Question: should we remove this axis as the cub radix sort (being used for `Types`) is always stable anyway?